source/Helpers/POETradeAPI/Models/TradeData/ExchangeMap.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+
+namespace Sidekick.Helpers.POETradeAPI.Models.TradeData
+{
+    public static class ExchangeMap
+    {
+        public static readonly Dictionary<string, string> Map = new Dictionary<string, string>()
+        {
+            { "orb-of-alteration", "alt" },
+            { "orb-of-fusing", "fuse" },
+            { "orb-of-alchemy", "alch" },
+            { "chaos-orb", "chaos" },
+            { "gemcutters-prism", "gcp" },
+            { "exalted-orb", "exa" },
+            { "chromatic-orb", "chrom" },
+            { "jewellers-orb", "jew" },
+            { "orb-of-chance", "chance" },
+            { "cartographers-chisel", "chisel" },
+            { "orb-of-scouring", "scour" },
+            { "blessed-orb", "blessed" },
+            { "orb-of-regret", "regret" },
+            { "regal-orb", "regal" },
+            { "divine-orb", "divine" },
+            { "vaal-orb", "vaal" },
+            { "scroll-of-wisdom", "wis" },
+            { "portal-scroll", "port" },
+            { "armourers-scrap", "scr" },
+            { "blacksmiths-whetstone", "whe" },
+            { "glassblowers-bauble", "ba" },
+            { "orb-of-transmutation", "tra" },
+            { "orb-of-augmentation", "aug" },
+            { "mirror-of-kalandra", "mir" },
+            { "eternal-orb", "ete" },
+            { "perandus-coin", "p" },
+            { "silver-coin", "silver" },
+            { "simple-sextant", "apprentice-sextant" },
+            { "prime-sextant", "journeyman-sextant" },
+            { "awakened-sextant", "master-sextant" },
+            { "blessing-of-xoph", "blessing-xoph" },
+            { "blessing-of-tul", "blessing-tul" },
+            { "blessing-of-esh", "blessing-esh" },
+            { "blessing-of-uul-netol", "blessing-uul-netol" },
+            { "blessing-of-chayula", "blessing-chayula" },
+            { "splinter-of-xoph", "splinter-xoph" },
+            { "splinter-of-tul", "splinter-tul" },
+            { "splinter-of-esh", "splinter-esh" },
+            { "splinter-of-uul-netol", "splinter-uul-netol" },
+            { "splinter-of-chayula", "splinter-chayula" },
+            { "sacrifice-at-dusk", "dusk" },
+            { "sacrifice-at-midnight", "mid" },
+            { "sacrifice-at-dawn", "dawn" },
+            { "sacrifice-at-noon", "noon" },
+            { "mortal-grief", "grie" },
+            { "mortal-rage", "rage" },
+            { "mortal-hope", "hope" },
+            { "mortal-ignorance", "ign" },
+            { "ebers-key", "eber" },
+            { "yriels-key", "yriel" },
+            { "inyas-key", "inya" },
+            { "volkuurs-key", "volkuur" },
+            { "fragment-of-the-hydra", "hydra" },
+            { "fragment-of-the-phoenix", "phoenix" },
+            { "fragment-of-the-miontaur", "minot" },
+            { "fragment-of-the-chimera", "chimer" },
+            { "oferring-to-the-goddess", "offer" }
```

Comment:
typo?

Reply:
The ExchangeMap is not necessary if I'm not wrong, you can just map the Item name with TradeClient.StaticItemCategories which contains all mappings for the items.
View here: https://www.pathofexile.com/api/trade/data/static

Reply:
@vestakip https://i.imgur.com/Swyzu4W.png